### PR TITLE
support `--repo` CLI option

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: bin/gh-pages [hunk 1]
```diff
@@ -10,12 +10,16 @@ program
       'base directory for all source files')
   .option('-s, --src <src>',
       'pattern used to select which files should be published', '**/*')
+  .option('-r, --repo <repo>',
+      'URL of the repository you\'ll be pushing to',
+      'URL of the `origin` remote of the current directory')
```
Comment (benoitvallon) [inline bin/gh-pages]: You don't need a third arg here otherwise it will be used as a default value if --repo is not specified

**Comment by benoitvallon** [comment]
:+1: having this new arg would allow us to do this like:

`gh-pages --repo https://$GH_TOKEN@github.com/user/private-repo.git -d dist`

File: bin/gh-pages (hunk 1 above)
Comment (cvan) [inline bin/gh-pages]: oops, don't know how that happened

**Comment by vvo** [comment]
cc @tschaub can we get this merged and bumped you think?

**Comment by cvan** [comment]
updated; good to go now?

**Comment by tschaub** [comment]
Thanks for the contribution @cvan!

**Comment by tschaub** [comment]
Published as `gh-pages@0.6.0`.

**Pull request merged**